TeachingRecordSystem/src/TeachingRecordSystem.Core/Jobs/InductionStatusUpdatedSupportJob.cs

```diff
@@ -0,0 +1,63 @@
+using System.Text.Json;
+using TeachingRecordSystem.Core.DataStore.Postgres;
+using TeachingRecordSystem.Core.Dqt;
+using TeachingRecordSystem.Core.Dqt.Queries;
+
+namespace TeachingRecordSystem.Core.Jobs;
+public class InductionStatusUpdatedSupportJob(TrsDbContext dbContext, ICrmQueryDispatcher crmQueryDispatcher, IClock clock)
+{
+    public const string JobSchedule = "0 3 * * *";  //3AM every day
+    public const string LastRunDateKey = "LastRunDate";
+
+    public async Task ExecuteAsync(CancellationToken cancellationToken)
+    {
+        var item = await dbContext.JobMetadata.SingleAsync(i => i.JobName == nameof(InductionStatusUpdatedSupportJob));
```

Comment:
Need to handle the first time this is run (when there's no metadata)
```suggestion
        var item = await dbContext.JobMetadata.SingleOrDefaultAsync(i => i.JobName == nameof(InductionStatusUpdatedSupportJob));
```